### PR TITLE
updated the front page of Sherpa RTD with references

### PR DESCRIPTION
**Pull request by anetasie**
I included the references to the three papers on the front page of Sherpa documentation.

### CODE REVIEW

**Comment by DougBurke** [comment]
Does anything need to get added to https://github.com/sherpa/sherpa/blob/main/README.md ?

**Comment by anetasie** [comment]
@DougBurke update README in #2154

File: docs/index.rst [hunk 1]
```diff
@@ -42,6 +43,18 @@ Information on recent releases and citation information for
...
+Acknowledging or Citing Sherpa
+==============================
+
+If you use Sherpa for work/research presented in a publication please cite the Sherpa papers:
+
+`Sherpa Paper 2024 <https://arxiv.org/abs/2409.10400>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2024arXiv240910400S/exportcitation>`_)
+
+`Sherpa Paper 2007 <https://articles.adsabs.harvard.edu/full/2007ASPC..376..543D>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2007ASPC..376..543D/exportcitation>`_)
+
+`Sherpa Paper 2001 <https://arxiv.org/abs/astro-ph/0108426>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2001SPIE.4477...76F/exportcitation>`_)
```
Comment (hamogu) [inline docs/index.rst:55]: The readthedocs build fails with 
```
/home/docs/checkouts/readthedocs.org/user_builds/sherpa/checkouts/2153/docs/index.rst:47: WARNING: Duplicate explicit target name: "ads bibtex".
/home/docs/checkouts/readthedocs.org/user_builds/sherpa/checkouts/2153/docs/index.rst:47: WARNING: Duplicate explicit target name: "ads bibtex".
```

I think (but did not test) that making the anonymous links (without an explicit entry in sphinx directory of targets shuld work):
```suggestion
`Sherpa Paper 2024 <https://arxiv.org/abs/2409.10400>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2024arXiv240910400S/exportcitation>`__)

`Sherpa Paper 2007 <https://articles.adsabs.harvard.edu/full/2007ASPC..376..543D>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2007ASPC..376..543D/exportcitation>`__)

`Sherpa Paper 2001 <https://arxiv.org/abs/astro-ph/0108426>`_ (`ADS BibTex <https://ui.adsabs.harvard.edu/abs/2001SPIE.4477...76F/exportcitation>`__)
```

**Review by hamogu — CHANGES_REQUESTED** [review]

**Comment by anetasie** [comment]
@hamogu it worked and the RTD page looks good.

**Review by hamogu — APPROVED** [review]

**Pull request merged**